gst/nnstreamer/tensor_transform/tensor_transform.c

```diff
@@ -947,10 +950,20 @@ gst_tensor_transform_set_property (GObject * object, guint prop_id,
       filter->mode = g_value_get_enum (value);
       break;
     case PROP_OPTION:
+    {
+      gchar *backup_option = filter->option;
       filter->option = g_value_dup_string (value);
       silent_debug ("Option = %s\n", filter->option);
-      gst_tensor_transform_set_option_data (filter);
+      if (TRUE == gst_tensor_transform_set_option_data (filter)) {
```

Comment:
What's the policy of `silent_debug ()`? If it has the purpose to trace state changes, I think this debug message should be moved inside this `if-statement`.

Reply:
Will change its behaviors.